.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Summary
+
+<!-- Please reference the issue this PR addresses in the following list. -->
+Addresses issue:
+
+- #
```

Comment:
I'm not sure this is exactly how we want the format here. For example, most PRs fix an issue, and to simplify maintenance it may be a better approach to have _Fixes #..._ by default. Potentially there could be a comment above that includes "If this PR does not fix the entire issue, change this to _Addresses #..._ instead."

Anyway, I think this needs a bit more definition and doesn't really belong in here.

Reply:
Updated.